index.html

```diff
@@ -7662,6 +7662,28 @@ <h3>Protocol Bindings</h3>
               present) accepted by the <a>Thing</a> in an interaction.
             </span>
           </li>
+          <li>
+            <span class="rfc2119-assertion" id="td-default-binding-method">
+              When no method (or similar) is indicated in a form,
+              <a>Default Value</a> MUST be assumed from the identified <a>Protocol Binding</a> from the form.
+            </span>
+            Please see
+            <a href="https://w3c.github.io/wot-binding-templates/bindings/protocols/http/#example-1">the example</a> in
+            the HTTP Binding.
+          </li>
+          <!-- TODO: This is not a behavior assertion. Where should the normative binding content go? -->
+          <li>
+            <span class="rfc2119-assertion" id="td-binding-multiple-ops">
+              In the case of a <code>forms</code> entry that has multiple <code>op</code> values, the form MUST NOT
+              restrict the messages that can be sent by the Consumer to only one operation. </span
+            >. For example, the form cannot contain <code>htv:methodName</code> in the case of the HTTP Binding.
+            Typically, a <a>TD Processor</a> will internally extend the multiple <code>op</code> values to separate
+            <code>forms</code> entries and associate each operation to the concrete protocol request using the default
```

Comment:
```suggestion
            <code>forms</code> entries and associate each operation with the concrete protocol request using the default
```

Not sure, with or to ?